core/embed/rust/src/translations/blob.rs

```diff
@@ -60,6 +60,28 @@ fn validate_offset_table(
     Ok(())
 }
 
+pub struct KerningTable<'a> {
+    triplets: &'a [(u16, u16, i8, u8)],
+}
+
+impl<'a> KerningTable<'a> {
+    pub fn new(mut reader: InputStream<'a>) -> Result<Self, Error> {
+        let count = reader.read_u16_le()?;
+        let triplets_len: usize = count.into();
+        let triplets_data =
+            reader.read((triplets_len / 6) * mem::size_of::<(u16, u16, i8, u8)>())?;
```

Comment:
Not sure why `triplets_len` is divided by 6 here...
Since it counts the number of triplets, the `triplets_data` size should be `triplets_len * 6` - right?